### PR TITLE
Add missing release notes entry for https://github.com/rails/rails/commit/f02a35b86efea24f1e2ab684bc8081ced5eb3b1a [ci skip]

### PR DESCRIPTION
**Pull request by prathamesh-sonpatki**

### CODE REVIEW

File: guides/source/5_0_release_notes.md [hunk 1]
```diff
@@ -499,6 +499,10 @@ Please refer to the [Changelog][action-view] for detailed changes.
     `datetime-local`.
     ([Pull Request](https://github.com/rails/rails/pull/25469))
 
+*   Make it possible to use blocks while rendering partials directly with
+    the `render partial:` helper.
```
Comment (kaspth) [inline guides/source/5_0_release_notes.md]: `Allow blocks while rendering with the …`?

Comment (prathamesh-sonpatki) [inline guides/source/5_0_release_notes.md]: @kaspth Thanks, made the change

**Comment by kaspth** [comment]
Thanks! 5-0-stable @ 46e0a69

**Pull request merged**